tests/python/rest_api/test_rq_job_dependencies.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (C) 2023 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from multiprocessing.pool import ThreadPool
+
+import pytest
+from redis import Redis
+from rq import Queue
+
+from shared.utils.helpers import generate_image_files
+
+from .utils import create_task
+
+
+@pytest.mark.usefixtures("restore_db_per_function")
+@pytest.mark.usefixtures("restore_cvat_data")
+@pytest.mark.usefixtures("restore_redis_db_per_function")
+class TestRqJobDependencies:
+    _USERNAME = "admin1"
+
+    def _create_task(self, idx: int) -> int:
+        task_spec = {
+            "name": f"Test task {idx}",
+            "labels": [
+                {
+                    "name": "car",
+                }
+            ],
+        }
+
+        task_data = {
+            "image_quality": 75,
+            "client_files": generate_image_files(3),
+        }
+
+        task_id, _ = create_task(self._USERNAME, task_spec, task_data)
+        return task_id
+
+    def test_check_rq_job_dependencies_after_creating_tasks_in_parallel(self, request):
+        if not request.session.config.getoption("--adjusted-server-conf"):
```

Comment:
The parameter name is too generic now to make such decisions as it is done here. Could you try to make it more clear and focused?